tests/pom.xml

```diff
@@ -252,7 +252,7 @@ Test cases are run against Docker images of Hono server + (Apache Qpid Dispatch
                     </log>
                     <wait>
                       <log>^.*(Started \w*Application in).*$</log>
-                      <time>30000</time>
+                      <time>40000</time>
```

Comment:
Oh yes, will move it out of this PR again (will need an increase though, often my environment needs about 31 seconds).